Cargo.toml

```diff
@@ -36,7 +36,7 @@ name = "makers"
 path = "src/makers.rs"
 
 [dependencies]
-ci_info = "^0.2.1"
+ci_info = { git = "https://github.com/sagiegurari/ci_info.git", features = ["serde-1"]}
```

Comment:
if you rebase you will see you are now on top of 0.3 so this can be modified to just add the features attribute.